core-ui/src/routing/clusterResources/addonsConfigurationCluster.routes.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { createResourceRoutes } from '../common';
+
+const List = React.lazy(() =>
```

Comment:
Is there a chance we can somehow automate it? We have now many files that look exactly the same, just point to a different file (that should probably have the same name as its route)